.eslintrc

```diff
@@ -5,9 +5,18 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended"
+    "plugin:@typescript-eslint/recommended",
   ],
   "rules": {
-    "max-len": ["error", { "code": 180, "comments": 180 }]
-  }
+    "max-len": ["error", { "code": 180, "comments": 180 }],
+    "@typescript-eslint/ban-types": [
+      "error",
+      {
+        "extendDefaults": true,
+        "types": {
+          "{}": false,
+        },
+      },
+    ],
```

Comment:
this is disabling the error when defining a type as "empty object" (= `{}`)

Reply:
can we add a comment here? I think eslintrc supports it and, if not, we could switch this to `.eslintrc.js`
https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats